lib/rubocop/cop/lint/top_level_return_with_argument.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop checks for top level return with arguments. If there is a
+      # top-level return statement with an argument, then the argument is
+      # always ignored. This is detected automatically since Ruby 2.7.
+      #
+      # This cop works by validating the ancestors of the return node. A
+      # top-level return node's ancestors will never belong to `AST::BlockNode`
+      # or `AST::DefNode` class.
+      #
```

Comment:
Please update and put inside the class so it doesn't show in the doc

Reply:
Updated this and added it inside class.